File: js/people_bus.js
var number = function(busStops){
	let passengers = 0;
	for (let i = 0; i < busStops.length; i++) {
		let gotIn = busStops[i][0];
		let gotOut = busStops[i][1];
		passengers = passengers + gotIn - gotOut;
		}
	return passengers
}

File: csharp/people_bus.cs
using System;
using System.Collections.Generic;

public class Kata
	{
		public static int Number(List<int[]> peopleListInOut)
		{
			int passengers = 0;
			
			foreach (var el in peopleListInOut)
			{
				int gotIn = el[0];
				int gotOut = el[1];
				passengers = passengers + gotIn - gotOut;
			}

			return passengers;
		}
	}